lib/sles4sap/console_redirection.pm

```diff
@@ -175,6 +175,11 @@ from serial console. VM ID is collected by opening 'log-console' which is not re
 sub check_serial_redirection {
     # Do not select serial console if it is already done. This avoids log pollution and speeds up process.
     if (is_serial_terminal()) {
+        # Teminate the process when scripts timed out
+        unless (wait_serial($testapi::distri->{serial_term_prompt})) {
```

Comment:
@lpalovsky  Thanks for your offline comments about the `console detection failures`, line 179 was updated and here are the VRs (no this failure now).
https://openqaworker15.qa.suse.cz/tests/298286#
https://openqaworker15.qa.suse.cz/tests/298287#

The original line is `unless (wait_serial(serial_term_prompt())) {`
In `lib/serial_terminal.pm` the `global $serial_term_prompt` is not reset/invoked in sles4sap lib file.
FYI:
In lib/serial_terminal.pm:
```
 sub serial_term_prompt {
    return $serial_term_prompt;
}
```
lib/sles4sap/console_redirection.pm:
```
sub set_serial_term_prompt {
    $testapi::distri->{serial_term_prompt} = '';    # This resets prompt since you don't know what user is currently logged in.
    $testapi::distri->{serial_term_prompt} = script_run('whoami | grep root', quiet => 1) == 0 ? '# ' : '> ';
}
```